src/monsters.cpp

```diff
@@ -82,11 +82,72 @@ ConditionDamage* Monsters::getDamageCondition(ConditionType_t conditionType, int
 {
 	ConditionDamage* condition =
 	    static_cast<ConditionDamage*>(Condition::createCondition(CONDITIONID_COMBAT, conditionType, 0, 0));
-	condition->setParam(CONDITION_PARAM_TICKINTERVAL, tickInterval);
-	condition->setParam(CONDITION_PARAM_MINVALUE, minDamage);
-	condition->setParam(CONDITION_PARAM_MAXVALUE, maxDamage);
-	condition->setParam(CONDITION_PARAM_STARTVALUE, startDamage);
 	condition->setParam(CONDITION_PARAM_DELAYED, 1);
+
+	uint32_t defaultTickInterval = 2000;
+	uint32_t defaultTickInterval2 = 0;
+	ConditionDamageFormula_t damageType = CONDITION_DAMAGE_FORMULA_CONSTANT;
+	int32_t rounds = 0;
+	int32_t damagePerRound = 0;
+
+	if (conditionType == CONDITION_FIRE) {
+		defaultTickInterval = 9000;
+		damageType = CONDITION_DAMAGE_FORMULA_CONSTANT;
```

Comment:
always CONDITION_DAMAGE_FORMULA_CONSTANT

Reply:
yeah, I wanted to be explicit for each condition to make it easier to read but I can remove them if its not prefer.